src/views/manage/list/listView.html

```diff
@@ -24,7 +24,10 @@
             </thead>
             <tbody>
             <tr ng-repeat="appointment in filteredAppointments" ng-click="select(appointment)" ng-style="{'background-color': colorsForListView[appointment.status]}" ng-class="{'selected' : isSelected(appointment)}">
-                <td>{{::appointment.patient.identifier}}</td>
+                <td>
+                    <a ng-show="shouldShowPatientDashboardHref" href="#" target="_blank" ng-click="$event.stopPropagation()" ng-href="{{patientDashboardURL + '?patientId=' + appointment.patient.uuid}}">{{::appointment.patient.identifier}}</a>
```

Comment:
And so the alias substitution will probably happen in the controller, I guess. But at least my point is that at this point the only thing that would happen is a generic alias substitution, and not the actual construction of an URL.